### PR TITLE
Quickstart instructions

### PR DESCRIPTION
**Pull request by aeijdenberg**
Added Quickstart instructions that work.  I think it's very important that anyone getting our code has an easy experience getting their first build and tests to succeed. 

Can we please try to keep these up to date as new dependencies are added?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,6 +5,65 @@ certificate-transparency
...
+First, install packaged dependencies:
+
+    sudo apt-get update -qq
+    sudo apt-get install -qq unzip cmake g++ libevent-dev golang-go autoconf pkg-config \
+        libjson-c-dev libgflags-dev libgoogle-glog-dev libprotobuf-dev libleveldb-dev \
+        libssl-dev libgoogle-perftools-dev protobuf-compiler libsqlite3-dev ant openjdk-7-jdk \
+        libprotobuf-java python-gflags python-protobuf python-ecdsa python-mock \
+        python-httplib2 git libldns-dev
+
+Next, we need `libevhtp` version `1.2.10` which is not yet in the Ubuntu packages, so we build from source:
```
Comment (pphaneuf) [inline README.md]: I'd say "which is not packaged in Ubuntu yet, so we build from source" here.

**Comment by pphaneuf** [comment]
Could you also flatten this into a single commit, and rebase on top of the latest `master`?

**Comment by aeijdenberg** [comment]
Done as suggested (I think).

PTAL.

**Comment by pphaneuf** [comment]
LGTM

**Pull request merged**